libs/components/data-manager/src/lib/modules/shared/sky-data-manager-resources.module.ts

```diff
@@ -24,11 +24,23 @@ const RESOURCES: { [locale: string]: SkyLibResources } = {
     },
     skyux_data_manager_apply_changes_button_title: { message: 'Apply changes' },
     skyux_data_manager_cancel_button_title: { message: 'Cancel' },
+    skyux_data_manager_select_all_button_aria_label: {
+      message: 'Select all {0}',
```

Comment:
Shouldn't these new resource strings include descriptions?

Reply:
Nope. This is the generated module that comes from the assets file that includes the descriptions.